technical-doc/grpc.rst

```diff
@@ -28,3 +28,11 @@ Integrations
 **Postman**: is a tool for software developers to create, test and debug API requests. More details can be found in
 `Postman learning center <https://learning.postman.com/docs/getting-started/introduction/>`_.
 Find all maintained Massa Postman collections in our official `workspace <https://www.postman.com/massalabs>`_.
+
+**Code generation**: See developer `documentation <https://github.com/massalabs/massa/blob/main/massa-grpc/README.md>`_.
```

Comment:
doesn't work for now, the file isn"t in main

Reply:
Yes 🙌, It's an anticipation from me hihih 🔗